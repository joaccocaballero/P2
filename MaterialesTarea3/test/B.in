insertarEnAbb (20, 20.0)
raiz
izquierdo
derecho
menorEnAbb
mayorEnAbb
buscarSubarbol 20
linealizacion
esVacioAbb
imprimirAbb
copiaAbb
consAbb (25, 25.0) 10 30
esPerfecto
menores 20
menores 40
caminoAscendente 20 1
removerDeAbb 20
esVacioAbb
insertarPalabra palas
imprimirPalabrasCortas 6
Fin