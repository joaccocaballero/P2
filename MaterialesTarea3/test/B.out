1>Se insertó (20,20.00) en el abb.
2>La raíz del abb es: (20,20.00)
3>El abb no tiene hijo izquierdo.
4>El abb no tiene hijo derecho.
5>El menor en el abb es: (20,20.00)
6>El mayor en el abb es: (20,20.00)
7>(20,20.00)
8>(20,20.00)
9>El abb NO es vacío.
10>
(20,20.00)
11>
(20,20.00)
12>
(25,25.00)
13>El abb es perfecto.
14>
15>
(20,20.00)
16>20 
La posición actual NO está definida.
17>Se removió 20 del abb.
18>El abb es vacío.
19>Insertando palas.
20>
palas
21>Fin.
